api/migrations/versions/0199_change_provider_details.py

```diff
@@ -0,0 +1,36 @@
+"""
+
+Revision ID: 0199
+Revises: 0198
+Create Date: 2018-09-10 09:34:40.184871
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0199'
+down_revision = '0198'
+
+
+def upgrade():
+    op.execute("UPDATE provider_details SET supports_international=True WHERE identifier='twilio'")
+    op.execute("UPDATE provider_details_history SET supports_international=True WHERE identifier='twilio'")
+    op.execute("UPDATE provider_details SET priority=15 WHERE identifier='telstra'")
+    op.execute("UPDATE provider_details_history SET priority=15 WHERE identifier='telstra'")
```

Comment:
this history table is like a log; semantically we should create a new entry for this rather than updating existing ones.

We'll wipe out all of our admin update history (not that there is much) if we go ahead with this

It's basically the same as  "provider_details" without the unique id